src/main/java/org/reactome/addlinks/fileprocessors/HPAFileProcessor.java

```diff
@@ -0,0 +1,53 @@
+package org.reactome.addlinks.fileprocessors;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class HPAFileProcessor extends FileProcessor<List<String>>
+{
+
+    public HPAFileProcessor(String processorName)
+    {
+        super(processorName);
+    }
+
+    public HPAFileProcessor()
+    {
+        super(null);
+    }
+
+    @Override
+    public Map<String, List<String>> getIdMappingsFromFile()
+    {
+        Map<String, List<String>> mappings = new HashMap<>();
+        try {
+            String dirToHPAFiles = this.unzipFile(this.pathToFile);
+            String inputFilepath = dirToHPAFiles + "/" + this.pathToFile.getFileName().toString().replace(".zip", "");
```

Comment:
Could `Paths.get()` be used here instead to avoid hard-coding the directory separator?